### PR TITLE
test: fix e2e-tests

### PR DESCRIPTION
**Pull request by sainoe**
Fix minor errors and bump Hermes version to v1.10.2 due a parsing error while relaying VSC packets.

### CODE REVIEW

File: tests/e2e/testnet-scripts/start-chain.sh [hunk 1]
```diff
@@ -375,16 +375,18 @@ fi
...
-    ""}},"block":null}'; do sleep 0.3 ; done
-    set -e
-fi
+# if [[ "$USE_COMETMOCK" == "true" ]]; then
+#     set +e
+#     until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
+#     set -e
+# else
+#     set +e
+#        until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658" | grep -q -v '{"block_id":{"hash":"","parts":{"total":0,"hash":
```
Comment (insumity) [inline tests/e2e/testnet-scripts/start-chain.sh:384]: The `height must be greater than 0` error seems to stem from this [bump](https://github.com/cosmos/interchain-security/pull/2200). I think we could have it `--type=height 1` and keep it as it was.

**Review by insumity — APPROVED** [review]
Great work!

File: tests/e2e/testnet-scripts/start-chain.sh (hunk 1 above)
Comment (sainoe) [reply tests/e2e/testnet-scripts/start-chain.sh]: This fixes the issue. Thanks!

File: tests/e2e/testnet-scripts/start-chain.sh [hunk 2]
```diff
@@ -377,15 +377,16 @@ fi
 # poll for chain start
 if [[ "$USE_COMETMOCK" == "true" ]]; then
     set +e
-    until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
+    until $BIN query block --type=height 1--node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
```
Comment (MSalopek) [inline tests/e2e/testnet-scripts/start-chain.sh:380]: This was printing a panic when you use 0, now you're waiting for the first block, right?

**Review by MSalopek — APPROVED** [review]
Admin approval

**Pull request merged**